spec.html

```diff
@@ -32989,13 +32987,11 @@
 
       <!-- es6num="24.1.2.1" -->
       <emu-clause id="sec-arraybuffer-length">
-        <h1>ArrayBuffer ( _length_ )</h1>
-        <p>ArrayBuffer called with argument _length_ performs the following steps:</p>
+        <h1>ArrayBuffer ( [ _length_ ] )</h1>
```

Comment:
But the `length` property should describe the typical number of arguments, and ArrayBuffers are seldom called with no arguments. [19.2.4.1](http://tc39.github.io/ecma262/#sec-function-instances-length):
> The value of the length property is an integer that indicates the typical number of arguments expected by the function.